cmd/vic-machine/list/list.go

```diff
@@ -114,11 +114,10 @@ func (l *List) prettyPrint(op trace.Operation, cli *cli.Context, vchs []*vm.Virt
 			version = vchConfig.Version.ShortVersion()
 			upgradeStatus = l.upgradeStatusMessage(op, vch, installerVer, vchConfig.Version)
 		}
-
-		parentPath := path.Dir(path.Dir(vch.InventoryPath))
-		name := path.Base(vch.InventoryPath)
+		// When the VCH was found the inventory path was overwritten with the resource pool path, so
+		// to print the path to the pool we need to call Dir twice.
```

Comment:
If we're stripping off two path elements I assume that's vch-pool/vch-endpointVM, meaning the InventoryPath would need to be for the endpointVM, not a resource pool. Is the comment about inventory path being overwritten with a pool path incorrect or am I reading it wrong?

Reply:
When the VM inventory path was over written it was replaced with `path.Join(pool.inventoryPath,vm.Name())` -- so there would be a stuttering of pool name and VCH name since they'll match.   So we strip off those two elements to provide the resource pool path.  This is how it works currently and I maintained that functionality.  

The VM inventory path is folder based, so we over write it with the resource pool inventory so we can provide the aforementioned functionality.  Otherwise the path would be folder based..i.e. `vcqaDC/vm` ..